### ISSUE TITLE
Fix test_create_errata_filter

### ISSUE THREAD

**Issue by sghai** [SatelliteQE/robottelo#2146]
Stacktrace
```
  File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel71-ui/tests/foreman/ui/test_contentviews.py", line 319, in test_create_errata_filter
    content_type, filter_type)
  File "robottelo/ui/contentviews.py", line 497, in add_filter
    common_locators["name"].send_keys(filter_name))
'\'tuple\' object has no attribute \'send_keys\'\n-------------------- >> begin captured logging << 
```

**Comment by sghai** [comment]
Following tests are also failing with same issue:

`test_create_package_filter` 
`test_create_package_group_filter`
`test_remove filter`

**Comment by sghai** [comment]
Fixed in PR #2154

**Comment by sghai** [comment]
Closing it.. PR got merged now
